docs/20_smart-contracts/10_getting-started/10_dune-guide/index.md

```diff
@@ -0,0 +1,634 @@
+info  nodeos    beast_http_listener.hp:79     listen               ] acceptor_.listen()
+info  nodeos    http_plugin.cpp:483           add_handler          ] add api url: /v1/node/get_supported_apis
+info  nodeos    producer_plugin.cpp:2534      produce_block        ] Produced block 014cab9772843d57... #2 @ 2023-02-17T00:16:44.500 signed by eosio [trxs: 0, lib: 1, confirmed: 0, net: 0, cpu: 100, elapsed: 20, time: 253]
+info  nodeos    producer_plugin.cpp:2534      produce_block        ] Produced block 1acdb1044c3362cd... #3 @ 2023-02-17T00:16:45.000 signed by eosio [trxs: 0, lib: 2, confirmed: 0, net: 0, cpu: 100, elapsed: 12, time: 311]
+info  nodeos    producer_plugin.cpp:2534      produce_block        ] Produced block 834ad8dc1f031c7c... #4 @ 2023-02-17T00:16:45.500 signed by eosio [trxs: 0, lib: 3, confirmed: 0, net: 0, cpu: 100, elapsed: 11, time: 365]
+info  nodeos    producer_plugin.cpp:2534      produce_block        ] Produced block 0639c4bbfb3b05a4... #5 @ 2023-02-17T00:16:46.000 signed by eosio [trxs: 0, lib: 4, confirmed: 0, net: 0, cpu: 100, elapsed: 15, time: 336]
+...
+```
+
+You can check that indeed the node is active and running by invoking the following command:
+
+```shell
+$ dune --list
+```
+
+The result lists all nodes created on the container, whether active or running, and their assigned ports:
+
+```
+Node Name   | Active? | Running? | HTTP           | P2P          | SHiP
+---------------------------------------------------------------------------------
+first       |    Y    |    Y     | 127.0.0.1:8888 | 0.0.0.0:9876 | 127.0.0.1:8080
+```
+
+You can also check that the node is listening to RPC API requests by running the following command:
+
+```shell
+$ dune -- cleos get info
+```
+
+The command returns information about the local EOS blockchain just launched:
+
+```json
+{
+  "server_version": "3f9cb99e",
+  "chain_id": "8a34ec7df1b8cd06ff4a8abbaa7cc50300823350cadc59ab296cb00d104d2b8f",
+  "head_block_num": 42987,
+  "last_irreversible_block_num": 42986,
+  "last_irreversible_block_id": "0000a7eac208ba842a890f28f8afa26a692760f1d824f618168f0bafa54ed28f",
+  "head_block_id": "0000a7ebb0dd4e78c5283ec67eb01a803673e23f77c57dee2733e1a170418dd9",
+  "head_block_time": "2023-02-17T06:14:57.000",
+  "head_block_producer": "eosio",
+  "virtual_block_cpu_limit": 200000000,
+  "virtual_block_net_limit": 1048576000,
+  "block_cpu_limit": 199900,
+  "block_net_limit": 1048576,
+  "server_version_string": "v3.2.1",
+  "fork_db_head_block_num": 42987,
+  "fork_db_head_block_id": "0000a7ebb0dd4e78c5283ec67eb01a803673e23f77c57dee2733e1a170418dd9",
+  "server_full_version_string": "v3.2.1-3f9cb99e068da8581036d212d74fb56144ef0fc4",
+  "total_cpu_weight": 0,
+  "total_net_weight": 0,
+  "earliest_available_block_num": 1,
+  "last_irreversible_block_time": "2023-02-17T06:14:56.500"
+}
+```
+
+You can also check the `get_info` endpoint directly exposed by the `chain_api_plugin` in your browser:
```

Comment:
Let's remove these parts as it isn't a part of DUNE specifically.

I also can't get it to work on windows using PowerShell setup

Reply:
Done